### PR TITLE
Packs/GSuiteAdmin.py: added user signout command

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by maimorag — COMMENTED** [review]
Hi @GalNakash-RecoLabs ,
Thank you for your contribution!
Good work :)

Few notes:
1.  Please bump the version of the pack in the pack_metadata.json file and add notes describing your changes to a new Release Notes file. To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

2. Please add your new command to the Test Playbook of the integration.
To do that, please upload the test-playbook to the XSOAR server, edit the playbook and add a task for testing your new command. When you are done - run the test-playbook to make sure it finishes successfully, if yes - download it, replace the YML of the old version of it with the new YML, and run the `demisto-sdk format` command on the YML. 
For more information about Test-Playbooks please see: https://xsoar.pan.dev/docs/integrations/test-playbooks.
For more information about the `demisto-sdk format` command please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#format.

3. Please add a unit test for your new command.
You can use the following guide - https://xsoar.pan.dev/docs/integrations/unit-testing as well as examples of unit tests in the HelloWorld integration and other packs from the content repository.

4. Please add the new coammnd to the readme file:  run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file and add -c and the coammnd name) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

**Comment by GalNakash-RecoLabs** [comment]
> Hi @GalNakash-RecoLabs , Thank you for your contribution! Good work :)
> 
> Few notes:
> 
> 1. Please bump the version of the pack in the pack_metadata.json file and add notes describing your changes to a new Release Notes file. To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
> 2. Please add your new command to the Test Playbook of the integration.
>    To do that, please upload the test-playbook to the XSOAR server, edit the playbook and add a task for testing your new command. When you are done - run the test-playbook to make sure it finishes successfully, if yes - download it, replace the YML of the old version of it with the new YML, and run the `demisto-sdk format` command on the YML.
>    For more information about Test-Playbooks please see: https://xsoar.pan.dev/docs/integrations/test-playbooks.
>    For more information about the `demisto-sdk format` command please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#format.
> 3. Please add a unit test for your new command.
>    You can use the following guide - https://xsoar.pan.dev/docs/integrations/unit-testing as well as examples of unit tests in the HelloWorld integration and other packs from the content repository.
> 4. Please add the new coammnd to the readme file:  run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file and add -c and the coammnd name) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

Hi I fixed everything, still have an issue with the pre-commit. Can you help with it?

**Comment by maimorag** [comment]
Hi @GalNakash-RecoLabs, pre-commit isn't a mandatory step here.
We're ready for a demo or a recorded video. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Comment by GalNakash-RecoLabs** [comment]
> 
Tomorrow 17:15?

**Comment by maimorag** [comment]
yes, great

**Comment by GalNakash-RecoLabs** [comment]
> yes, great

what's your email address? I will share an invite/

File: Packs/GoogleDrive/ReleaseNotes/1_2_36.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #### Integrations
 ##### Google Drive
+- Added use_domain_admin_access param to google-drive-file-permissions-list.
```
Comment (maimorag) [inline Packs/GoogleDrive/ReleaseNotes/1_2_36.md:3]:
```suggestion
- Added the *use_domain_admin_access* param to google-drive-file-permissions-list.
```

**Review by maimorag — COMMENTED** [review]
The code looks great, 
Please bump the pack version in the pack_metadata.json file in GoogleDrive pack to the release note number you've created.

**Comment by GalNakash-RecoLabs** [comment]
> The code looks great, Please bump the pack version in the pack_metadata.json file in GoogleDrive pack to the release note number you've created.

Version is aligned

File: Packs/GoogleDrive/Integrations/GoogleDrive/README.md [hunk 2]
```diff
@@ -1,39 +1,45 @@
 Google Drive allows users to store files on their servers, synchronize files across devices, and share files. This integration helps you to create a new drive, query past activity, and view change logs performed by the users.
-This integration was integrated and tested with version 1.31.0 of GoogleDrive
+This integration was integrated and tested with version xx of GoogleDrive
```
Comment (maimorag) [inline Packs/GoogleDrive/Integrations/GoogleDrive/README.md:2]:
```suggestion
This integration was integrated and tested with version 1.31.0 of GoogleDrive
```

**Review by maimorag — APPROVED** [review]

**Pull request merged**